src/main/java/com/myhome/controllers/PaymentController.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright 2020 Prathab Murugan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.myhome.controllers;
+
+import com.myhome.controllers.dto.PaymentDto;
+import com.myhome.controllers.mapper.SchedulePaymentApiMapper;
+import com.myhome.controllers.request.SchedulePaymentRequest;
+import com.myhome.controllers.response.ListMemberPaymentsResponse;
+import com.myhome.controllers.response.SchedulePaymentResponse;
+import com.myhome.domain.Community;
+import com.myhome.domain.CommunityAdmin;
+import com.myhome.domain.HouseMember;
+import com.myhome.domain.Payment;
+import com.myhome.services.CommunityService;
+import com.myhome.services.PaymentService;
+import io.swagger.v3.oas.annotations.Operation;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Wildcard imports.
https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

Please setup IntelliJ correctly
https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports